server/memcached/src/main/java/org/infinispan/server/memcached/MemcachedServer.java

```diff
@@ -76,6 +76,8 @@ public void stop() {
 
    @Override
    public int getWorkerThreads() {
-      return Integer.getInteger("infinispan.server.memcached.workerThreads", configuration.workerThreads());
+      //return Integer.getInteger("infinispan.server.memcached.workerThreads", configuration.workerThreads());
+      return 1; // Unused for now, so just return the smallest possible valid value
```

Comment:
IMO the end-of-line comment should be a regular comment, and we don't need the commented-out line. In fact, now that thread information is exposed separately, maybe we should deprecate `getWorkerThreads()`?